tools.sh

```diff
@@ -130,21 +130,22 @@ function get_vm_prefix {
 }
 
 function shutdown_vm {
-    local vm_prefix=$1
-    retry sudo virsh shutdown ${vm_prefix}-master-0
+    local vm_name=$1
+    retry sudo virsh shutdown ${vm_name}
     # Wait till instance started successfully
-    until sudo virsh domstate ${vm_prefix}-master-0 | grep shut; do
-        echo " ${vm_prefix}-master-0 still running"
+    until sudo virsh domstate ${vm_name} | grep shut; do
+        echo " ${vm_name} still running"
         sleep 3
     done
 }
 
 function start_vm {
-    local vm_prefix=$1
-    retry sudo virsh start ${vm_prefix}-master-0
+    local vm_name=$1
+    local vm_ip=$2
+    retry sudo virsh start ${vm_name}
     # Wait till ssh connection available
-    until ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- "exit 0" >/dev/null 2>&1; do
-        echo " ${vm_prefix}-master-0 still booting"
+    until ${SSH} core@${vm_ip} -- "exit 0" >/dev/null 2>&1; do
+        echo " ${vm_name} still booting"
```

Comment:
Might be worth configuring https://libvirt.org/nss.html at some point so that we don't need the IP in addition to the VM name. With libvirt-nss configured, we can just do `${SSH} core@${CRC_VM_NAME}`

Reply:
I am goingto create a followup issue for that instead of putting that effort right now. It is a good suggestion.